networking.go

```diff
@@ -1,10 +1,20 @@
 package dastard
 
-// TCP port numbers used by Dastard.
-const (
-	PortRPC int = 5500 + iota
-	PortStatus
-	PortTrigs
-	PortSecondaryTrigs
-	PortSummaries
-)
+// Portnumbers holds all TCP port numbers used by Dastard.
+type Portnumbers struct {
+	RPC            int
+	Status         int
+	Trigs          int
+	SecondaryTrigs int
+	Summaries      int
+}
+
+var Ports = Portnumbers{5500, 5501, 5502, 5503, 5504}
```

Comment:
This should be `setPortnumbers(5500)`? Can you call a function in global scope? I guess it would need to be in an `init` function, then it would run after any `TestMain` and default your goal, so I guess it should stay as you wrote it.

Reply:
You can't call `setPortnumbers(5500)` at the package level. It could go in every main program, but that would be easy to forget.

What you _can_ apparently do is define a `function init() {...}` in each Go file, which will executed when the program starts. Such a function cannot be called other than automatically. Cool, huh? I think I'll use this idea and update the PR.

Reply:
What out because I think `init` runs after `TestMain` which means it may be hard to override ports in tests reliably.